sources/models/shared-defaults/kubernetes-containerd.toml

```diff
@@ -1,3 +1,7 @@
 [configuration-files.containerd-config-toml]
 # No override to path
 template-path = "/usr/share/templates/containerd-config-toml_k8s"
+
+# Image registries
+[metadata.settings.image-registry]
+affected-services = ["containerd"]
```

Comment:
It is k8s-specific since it impacts cri-containerd, which is specific to k8s.

The containerd config file that gets rendered is `containerd-config-toml_k8s`, the generic one being `containerd-config-toml_basic`

Reply:
Ah, ok.  Do we need a similar block in `containerd-config-toml_basic` then, or is it only possible to use mirrors with the setup that k8s supports?

Reply:
It's a configuration only available to the `cri` containerd plugin. I don't believe containerd itself has a equivalent configuration for registry mirrors.

Reply:
With normal usage of containerd (without the cri-containerd plugin), image pull logic (including network transfer) happens in the client program, not in the containerd daemon.  Most client programs don't read containerd's config file.